pyproject.toml

```diff
@@ -358,10 +358,6 @@ lint.ignore = [  # NOTE: non-permanent exclusions should be added to `.ruff.toml
     # flake8-fixme (FIX)
     "FIX002",  # Line contains TODO | notes for improvements are OK iff the code works
 
-    # ISC001 shouldn't be used with ruff format
```

Comment:
I will defer to the other devs on this one since I don't follow the rule updates. Other lines LGTM. Thanks!